src/Tribe/Main.php

```diff
@@ -4874,11 +4802,69 @@ public function remove_preview_organizers( $event_id, $delete_meta = false ) {
 			}
 		}
 
+		/**
+		 * Fetches and verify if we had a delayed activation
+		 *
+		 * @deprecated TBD
+		 *
+		 * @since  4.7
+		 *
+		 * @return boolean [description]
+		 */
+		public function is_delayed_activation() {
+/*			_deprecated_function(
+				__METHOD__,
+				'',
+				'TBD'
```

Comment:
This _deprecated_function() code block is commented out—is that intentional? More work needed here? If not, remove it